src/Umbraco.Infrastructure/Persistence/Repositories/Implement/DocumentVersionRepository.cs

```diff
@@ -186,6 +180,31 @@ public void SetPreventCleanup(int versionId, bool preventCleanup)
             .On<UserDto, ContentVersionDto>(left => left.Id, right => right.UserId)
             .Where<ContentVersionDto>(x => x.Id == versionId);
 
-        return _scopeAccessor.AmbientScope?.Database.Single<ContentVersionMeta>(query);
+        ContentVersionMeta? result = _scopeAccessor.AmbientScope?.Database.Single<ContentVersionMeta>(query);
+        EnsureUtcDate(result);
+        return result;
+    }
+
+    private static void EnsureUtcDate(ContentVersionMeta? version)
+    {
+        if (version is null)
+        {
+            return;
+        }
+
+        EnsureUtcDates([version]);
+    }
```

Comment:
No need to allocate a new list.
```suggestion
    private static void EnsureUtcDate(ContentVersionMeta? version) => version?.EnsureUtc();
```